pkg/messaging/mqtt/pubsub.go

```diff
@@ -135,10 +140,10 @@ func (ps pubsub) Unsubscribe(id, topic string) error {
 	case true:
 		// Check topic
 		if ok := s.contains(topic); !ok {
-			return errNotSubscribed
+			return ErrNotSubscribed
 		}
 	default:
-		return errNotSubscribed
+		return ErrNotSubscribed
```

Comment:
You can simplify this `switch` with short-circuit evaluation:
```go
s, ok := ps.subscriptions[id]
if !ok || !s.contains(topic) {
	return ErrNotSubscribed
}
```
Also, remove unnecessary comments (_Check client id_ and _Check topic_)